src/ImageSequencer.js

```diff
@@ -292,6 +299,30 @@ ImageSequencer = function ImageSequencer(options) {
     });
   }
 
+  function loadNewModule(name,options){
+
+    if(!options) {
+      return this;
+    }
+    else if(Array.isArray(options)) {
```

Comment:
Here, i prefer `} else if (...) {` spacing, does that sound OK?

Reply:
Got it!!